docs/contributing/documentation.md

```diff
@@ -42,7 +42,7 @@ If possible, include an executable example showing the module using the `@exampl
 
 ## Class Level Documentation
 
-Class level documentation includes components and Higher-order Components (HOCs).  Each component or HOC export should have a class level block that discusses the features and provides an example usage. This block also serves as a place to indicate any HOCs that may be applied or to cross-reference related components.
+Class level documentation includes components and Higher-order components (HOCs).  Each component or HOC export should have a class level block that discusses the features and provides an example usage. This block also serves as a place to indicate any HOCs that may be applied or to cross-reference related components.
```

Comment:
The H should be lowercase everywhere (except where it starts a sentence, of course).